src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -384,11 +384,11 @@ public static class TreeExtensions
         }
 
         /// <summary>
-        /// Predict a target using a decision tree regression model trained with the <see cref="FastForestBinaryTrainer"/>.
+        /// Creates <see cref="FastForestBinaryTrainer"/>, which predicts a target using a decision tree regression model.
```

Comment:
Creates -> Create

Reply:
I did fix that, are you on iteration 6?

---
In reply to: [277064277](https://github.com/dotnet/machinelearning/pull/3399#discussion_r277064277) [](ancestors = 277064277)